Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -2456,7 +2463,9 @@ GRefPtr<GdkEvent> webkitWebViewBaseTakeContextMenuEvent(WebKitWebViewBase* webki
 #else
 GUniquePtr<GdkEvent> webkitWebViewBaseTakeContextMenuEvent(WebKitWebViewBase* webkitWebViewBase)
 {
-    return WTFMove(webkitWebViewBase->priv->contextMenuEvent);
+    if (webkitWebViewBase->priv->contextMenuEvent)
+        return WTFMove(webkitWebViewBase->priv->contextMenuEvent);
+    return dummyContextMenuEvent();
```

Comment:
I don't think this is right, we end up with a nullptr event because media controls are using context menu api to implement a popup menu. I don't think we should emit context-menu signal for the media controls settings menu.

Reply:
Good point